### ISSUE TITLE
Troubleshooting missing right click menu?

### ISSUE THREAD

**Issue by jdestgermain** [jupyterlab/jupyterlab-latex#57]
I've run all the steps in the readme, but I don't see the option on my `.tex` file to preview the file.

What are some steps I can do to troubleshoot?  

Windows 7, fresh Jupyter Lab installation (0.31.4).   `pip install jupyterlab_latex`, `jupyter labextension install @jupyterlab/latex` and I even tried `jupyter serverextension enable --sys-prefix jupyterlab_latex`

EDIT: MikTeX installed on my system, can successfully save notebooks to PDF which renders fine.

**Comment by ian-r-rose** [comment]
Hi @jdestgermain, can you check the Javascript console in your browser to see if any errors are showing? It would also be useful to check the log in the terminal where the notebook server is running to see if there are any errors that look related to `jupyterlab-latex`.

**Comment by jdestgermain** [comment]
Looks like it might be throwing an error about `jupyter_nbextensions_configurator`?

```jupyter lab
[W 09:32:26.215 LabApp] Error loading server extension jupyter_nbextensions_configurator
    Traceback (most recent call last):
      File "C:\ProgramData\Anaconda3\lib\site-packages\notebook\notebookapp.py", line 1451, in init_server_extensions
        mod = importlib.import_module(modulename)
      File "C:\ProgramData\Anaconda3\lib\importlib\__init__.py", line 126, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 994, in _gcd_import
      File "<frozen importlib._bootstrap>", line 971, in _find_and_load
      File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
    ModuleNotFoundError: No module named 'jupyter_nbextensions_configurator'
[I 09:32:26.224 LabApp] JupyterLab beta preview extension loaded from C:\ProgramData\Anaconda3\lib\site-packages\jupyterlab
[I 09:32:26.224 LabApp] JupyterLab application directory is C:\ProgramData\Anaconda3\share\jupyter\lab
[W 09:32:27.299 LabApp] JupyterLab server extension not enabled, manually loading...
[I 09:32:27.300 LabApp] JupyterLab beta preview extension loaded from C:\ProgramData\Anaconda3\lib\site-packages\jupyterlab
[I 09:32:27.300 LabApp] JupyterLab application directory is C:\ProgramData\Anaconda3\share\jupyter\lab
[I 09:32:28.506 LabApp] Serving notebooks from local directory: C:\workzones\iPython
[I 09:32:28.507 LabApp] 0 active kernels
[I 09:32:28.507 LabApp] The Jupyter Notebook is running at:
[I 09:32:28.507 LabApp] http://localhost:8888/?token=307ecb56155635bbeda52c29ac95cc847d7ae88fb9c99ab2
[I 09:32:28.507 LabApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 09:32:28.511 LabApp]

    Copy/paste this URL into your browser when you connect for the first time,
    to login with a token:
        http://localhost:8888/?token=307ecb56155635bbeda52c29ac95cc847d7ae88fb9c99ab2
[I 09:32:28.635 LabApp] Accepting one-time-token-authenticated connection from ::1
[I 09:32:30.274 LabApp] Build is up to date
```

In the javascript console I see this:
```
index.js:267 Plugin 'jupyterlab-latex:open' failed to activate.
(anonymous) @ index.js:267
09:32:29.231 index.js:268 Error: No provider for: @jupyterlab/docmanager:IDocumentManager.
    at t.e.resolveRequiredService (index.js:182)
    at index.js:140
    at Array.map (<anonymous>)
    at t.e.activatePlugin (index.js:140)
    at index.js:266
    at Array.map (<anonymous>)
    at t.e.start (index.js:265)
    at i (index.out.js:1234)
(anonymous) @ index.js:268
09:32:29.593 
```

**Comment by ian-r-rose** [comment]
It looks like the second error message is the problem. Can you tell me what versions of JupyterLab and `@jupyterlab/latex` you are running?

**Comment by jdestgermain** [comment]
```
C:\workzones\iPython>pip show jupyterlab_latex
Name: jupyterlab-latex
Version: 0.2.0
Summary: A Jupyter Notebook server extension which acts as an endpoint for LaTeX.
Home-page: http://jupyter.org
Author: Jupyter Development Team
Author-email: jupyter@googlegroups.com
License: BSD
Location: c:\programdata\anaconda3\lib\site-packages
Requires: notebook

C:\workzones\iPython>jupyter labextension list
JupyterLab v0.31.4
Known labextensions:
   app dir: C:\ProgramData\Anaconda3\share\jupyter\lab
@jupyterlab/latex
        @jupyterlab/latex v0.2.0 enabled  ok

C:\workzones\iPython>pip show jupyterlab
Name: jupyterlab
Version: 0.31.4
Summary: The JupyterLab notebook server extension.
Home-page: http://jupyter.org
Author: Jupyter Development Team
Author-email: jupyter@googlegroups.com
License: BSD
Location: c:\programdata\anaconda3\lib\site-packages
Requires: notebook, jupyterlab-launcher, ipython-genutils

C:\workzones\iPython>
```

**Comment by ian-r-rose** [comment]
Ah, JupyterLab v0.31.4 had a bug when it came to resolve the correct versions of packages for lab extensions. Can you upgrade to the most recent version of JupyterLab, rebuild (`jupyter lab build`), and try again?

**Comment by jdestgermain** [comment]
Which is the most recent version of JupyterLab?  I had installed using `conda install -c conda-forge jupyterlab` today.  Should I use `pip install jupyterlab` instead?

**Comment by jdestgermain** [comment]
It looks like uninstalling using conda and then re-installing using pip fixes it, as it upgrades to v0.31.8.

Thanks

**Comment by ian-r-rose** [comment]
Hmm, I am not sure why the conda-forge version was lagging, but glad it is working!
